src/GoTrueClient.ts

```diff
@@ -1040,23 +1040,30 @@ export default class GoTrueClient {
    * @param factorID System assigned identifier for authenticator device as returned by enroll
    * @param code Code Generated by an authenticator device
    */
-  private async _verify(params: MFAVerifyParams) {
+  private async _verify(params: MFAVerifyParams): Promise<AuthResponse> {
     const { data, error } = await this.getUser()
     if (error) throw error
     const user: User = data.user
     try {
-      return await _request(
+      const { data, error } = await _request(
         this.fetch,
         'POST',
         `${this.url}/user/${user.id}/factor/${params.factorId}/verify`,
         {
           body: { code: params.code, challenge_id: params.challengeId },
           headers: this.headers,
+          xform: _sessionResponse,
```

Comment:
What does is the xform property used for again?

Reply:
it acts as a callback function and is used to transform the API response returned into a desirable format